core/src/style/pointStyleBuilder.cpp

```diff
@@ -358,44 +323,28 @@ void PointStyleBuilder::labelPointsPlacing(const Line& _line, const glm::vec4& u
             }
             break;
         case LabelProperty::Placement::spaced: {
-            Line interpolatedLine;
-            std::vector<float> allAngles;
+            LineSampler<std::vector<Point>> sampler;
```

Comment:
yup make sense, I saw the use of `z` in LineSampler after writing this comment :D.